index.js

```diff
@@ -412,9 +412,7 @@ function removeAgentIfSuperseded(server, auth, agents, dryrun) {
 }
 
 var removeDisabledAgents = function(server, auth, dryrun) {
-  console.log("removing disabled agents")
```

Comment:
Why get rid of this one?

Reply:
Cos it was ugly and not-coloured :)

(and duplicated the one two lines below)